Style.md

```diff
@@ -107,6 +107,18 @@ in the reference, omit it.
 
 Spelled as two words, not closed up or hyphenated.
 
+## comments
+
+Prefer full sentences for comments in code.
+End with a period, even when it's a fragment.
+When contrasting code that works and code that doesn't,
+format the comments with a colon as follows:
```

Comment:
@colleenish Thoughts on adding this preference for full sentences?  TSPL is currently pretty mixed between sentence case vs all lower case, and fragments vs sentences.